api/v1alpha1/workloadconfig.go

```diff
@@ -0,0 +1,110 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	SchemeBuilder.Register(&WorkloadConfiguration{})
+	SchemeBuilder.Register(&WorkloadConfigurationList{})
+}
+
+// WorkloadConfiguration allows additional tuning of control plane workloads
```

Comment:
may be it's just me but "workloadconfig" makes me think of config for workloads running on the guest cluster.

Reply:
I also think this.  Workload is extremely generic.  We are configuring a cluster.  I like `HostedClusterConfiguration` personally.